sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql

```diff
@@ -243,10 +243,10 @@ aggregated_newtab_activity AS (
 client_profile_info AS (
```

Comment:
Are these multiple values due to profiles? -- I'd expect one per client_id.

Reply:
Ah, you're right. I copied that over from the existing query without thinking about it. Fixing it